innkeeper/client.go

```diff
@@ -334,7 +334,7 @@ func (c *Client) writeRoute(url string, route *routeData) error {
 		return err
 	}
 
-	req.Header.Add(authHeaderName, authToken)
+	req.Header.Add(authHeaderName, "Bearer " + authToken)
```

Comment:
How does innkeeper handle multiple `Authorization` headers? Should be a `.Set()` instead?